test/multi_node_test.py

```diff
@@ -298,5 +298,50 @@ def testConnectingInLocalCase(self):
         self.assertIn("success", out)
 
 
+class RunDriverForMultipleTimesTest(unittest.TestCase):
+    def tearDown(self):
+        ray.shutdown()
+
+    def testRunDriverForTwice(self):
+        # We used to have issue 2165 and 2288: driver will hang when we run it
```

Comment:
Can you include the full URL instead of just the issue number? This will make it easier to navigate to the issue.

Reply:
OK.